activerecord/lib/active_record/inheritance.rb

```diff
@@ -94,14 +94,22 @@ def finder_needs_type_condition? # :nodoc:
         :true == (@finder_needs_type_condition ||= descends_from_active_record? ? :false : :true)
       end
 
-      # Returns the class descending directly from ActiveRecord::Base, or
-      # an abstract class, if any, in the inheritance hierarchy.
+      # Returns the first class in the inheritance hierarchy that descends from either an
+      # abstract class or from ActiveRecord::Base
```

Comment:
```suggestion
      # abstract class or from +ActiveRecord::Base+.
```

Reply:
Oh actually, docs explicitly say to use `<tt></tt>` for `ActiveRecord::Base`, so let's use that instead: https://guides.rubyonrails.org/api_documentation_guidelines.html#fixed-width-font